cmd/agent/app/snmp.go

```diff
@@ -85,7 +85,7 @@ func init() {
 	snmpWalkCmd.Flags().StringVarP(&communityString, "community-string", "C", "", "Set the community string")
 
 	// snmp v3 specific
-	snmpWalkCmd.Flags().StringVarP(&authProt, "auth-protocol", "a", defaultAuthProtocol, "Set authentication protocol (MD5|SHA|SHA-224|SHA-256|SHA-384|SHA-512)")
+	snmpWalkCmd.Flags().StringVarP(&authProt, "auth-protocol", "a", defaultAuthProtocol, "Set authentication protocol (MD5|SHA|SHA224|SHA-224|SHA256|SHA-256|SHA384|SHA-384|SHA512|SHA-512)")
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Since we support both format now in the switch, we can keep the previous values in the helper. I wonder if having both could be confusing to the customer.

Reply:
should we keep only the hyphon values same as the `net-snmp` `snmpwalk` or should we keep the values without hyphens since that's what the protocol actually is ?

Reply:
Let's keep the ones with hyphens